frontend/src/components/LineChart/LineChart.js

```diff
@@ -0,0 +1,53 @@
+import { PropTypes } from 'prop-types'
+import { useMemo, useRef } from 'react'
+import { useTheme } from '@mui/system'
+import { VictoryAxis, VictoryChart, VictoryLine, VictoryScatter, VictoryTooltip } from 'victory'
+import Box from '@mui/material/Box'
+
+import { nFormatter } from 'utils/helpers'
+import { getLineChartDomain, useVictoryTheme, useClientSize } from 'utils/victory'
+import ChartFlyOut from 'components/ChartFlyOut'
+
+const LINE_CHART_RATIO = 0.75
+
+const LineChart = ({ data, title, xAxisLabel, yAxisLabel }) => {
+  const theme = useTheme()
+  const domain = useMemo(() => getLineChartDomain(data, LINE_CHART_RATIO), [data])
+  const victoryTheme = useVictoryTheme(theme)
+
+  const ref = useRef()
+  const size = useClientSize(ref)
+
+  return (
+    <Box ref={ref} sx={{ columns: 1, marginTop: '-50px' }}>
```

Comment:
Why are `columns` and `marginTop` needed here?

Reply:
specifying `columns` was required because tooltip pane is displayed in the right column assuming it has 2 columns inheriting parent value.